### ISSUE TITLE
Provide a built bundle via GitHub Release

### ISSUE THREAD

**Issue by BobVul** [warrenbuckley/Compact-Log-Format-Viewer#10]
Currently, [releases](https://github.com/warrenbuckley/Compact-Log-Format-Viewer/releases) only provide source bundles. The only way to get the app without using a Store (not possible on company/offline machines) is to build it from source. While that's not a problem for a dev, it makes it harder to recommend for others.

Please consider providing the AppX builds via an alternative source, e.g. GitHub Releases.

P.S. Is this project available under any specific licence?

**Comment by enkelmedia** [comment]
I agree! Don't want to login to a personal account on client machines etc. Would be great to have a "portable" version

**Comment by warrenbuckley** [comment]
I need to give this project some ❤️ tomorrow and will be creating a GitHub Action pipeline.
The reason to help try & push store approach as it deals with the auto updating of the app,

**Comment by lindeberg** [comment]
Would like this too! Tricky to use Windows Store on Windows Server.

**Comment by warrenbuckley** [comment]
OK the MSI installer and Windows Store .appx files are part of the latest release & will be for future ones too
https://github.com/warrenbuckley/Compact-Log-Format-Viewer/releases/tag/v1.1.0
